src/Chunk.cpp

```diff
@@ -741,6 +767,29 @@ void cChunk::MoveEntityToNewChunk(cEntity * a_Entity)
 
 
 
+void cChunk::UpdateCanUnload()
```

Comment:
I'm very unease about this function and the m_CanUnload member variable, but nevermind.